modules/oe_content_event/README.md

```diff
@@ -28,6 +28,16 @@ The `field_group` module requires the following patches to be applied:
 }
 ```
 
+The `inline_entity_form` module requires the following patches to be applied:
+
+```json
```

Comment:
modules/oe_content_project/README.md should be updated also with this text.

Reply:
https://github.com/openeuropa/oe_content/pull/262